OpenEdXMobile/res/layout/fragment_whats_new.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <FrameLayout
```

Comment:
This layout must use LinearLayout with vertical orientation to align top, bottom and center view in line. Making constraints based on margins is not good approach if screen design goes complex in future, specially if we need different margins on different densities.